adafruit_tinylora/ttn_cn.py

```diff
@@ -0,0 +1,37 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2018 Brent Rubell for Adafruit
```

Comment:
Please add your name here, you wrote this file :) 

or add a line underneath my name as `# Modified by IAMLIUBO, 2021`

Reply:
Add done. :)

Reply:
Thanks, you deserve credit! I'll wait for advice on resolving the CI issue and then merge this in.

Reply:
Haha, OK.
I thought it was a problem with my coding style, but it seems to be a problem with the CI script (I guess...
Have a nice day.
I need to rest.